src/graphql.server.ts

```diff
@@ -1,21 +1,19 @@
-import { Express } from "express";
-import { ApolloServer, gql } from "apollo-server-express";
+import { Express } from 'express'
+import 'reflect-metadata'
+import { ApolloServer, gql } from 'apollo-server-express'
+import { buildSchema } from 'type-graphql'
+import { resolvers } from './resolvers'
 
 export const buildGraphQLServer = async (app: Express) => {
-  const server = new ApolloServer({
-    typeDefs: gql`
-      type Query {
-        hello: String
-      }
-    `,
-    resolvers: {
-      Query: {
-        hello: () => "Hello GraphQL",
-      },
-    },
-  });
+	const schema = await buildSchema({
+		resolvers,
```

Comment:
add the option to generate the schema file too, please. I think it can generate the type definitions file too.

Reply:
https://typegraphql.com/docs/emit-schema.html

Reply:
check it out, please!